kurl_util/cmd/bashmerge/main.go

```diff
@@ -167,16 +167,16 @@ func parseBashFlags(installer *kurlv1beta1.Installer, bashFlags string) error {
 				installer.Spec.Kurl = &kurlv1beta1.Kurl{}
 			}
 			installer.Spec.Kurl.IgnoreRemoteUpgradePrompt = true
-		case "preflight-ignore":
```

Comment:
These flags are publicly documented.  We should keep them for backwards compatibility.